custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSearch.kt

```diff
@@ -109,4 +111,52 @@ class WorkOnSearch(
             }
         }
     }
+
+    private fun subscribeToFilter() {
+        measure.roll(subscribeToFilterProbability) {
+            val filter = filters.recall()!!
+            jira.navigateTo(filter.toString())
+            issueNavigator.waitForNavigator()
+            measure.measure(
+                key = MeasureType.SUBSCRIBE_TO_FILTER,
+                action = {
+                    measure.measure(
+                        key = MeasureType.SUBSCRIBE_TO_FILTER_DETAILS_WDW,
+                        action = {
+                            issueNavigator.clickOnDetails()
+                            issueNavigator.checkSubscriptions()
```

Comment:
clickOnDetails and checkSubscriptions could be merged - checkSubscriptions is part of clickOnDetails